code/modules/lighting/lighting_process.dm

```diff
@@ -1,35 +1,69 @@
+#define MAX_LIGHT_UPDATES_PER_WORK   100
+#define MAX_CORNER_UPDATES_PER_WORK  1000
+#define MAX_OVERLAY_UPDATES_PER_WORK 2000
+
 /datum/controller/process/lighting
+	schedule_interval = LIGHTING_INTERVAL
```

Comment:
That's just kinda the way the process scheduler is - schedule_interval is set to a default value in New(), and set to its actual value in setup(). Setting it here does nothing.